environment.yml

```
@@ -29,6 +30,8 @@ dependencies:
   - jupyter-book
   - jupyterlab
   - jupyter_server<2
```

```suggestion
  - jupyter_server
```
We removed the version pin from the cookbook template a while back. Best to remove it here to avoid potential env conflicts.